parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -347,10 +347,6 @@ MASTER_CONTAINER_ADDONS_PLACEHOLDER
     sed -i "s|<cloud>|{{WrapAsParameter "kubernetesClusterAutoscalerAzureCloud"}}|g; s|<useManagedIdentity>|{{WrapAsParameter "kubernetesClusterAutoscalerUseManagedIdentity"}}|g" /etc/kubernetes/addons/cluster-autoscaler-deployment.yaml
 {{end}}
 
-{{if and (not IsPrivateCluster) (eq .OrchestratorProfile.KubernetesConfig.LoadBalancerSku "Standard")}}
-    sed -i "s|<svcName>|{{WrapAsParameter "kuberneteselbsvcname"}}|g" "/etc/kubernetes/addons/elb-svc.yaml"
```

Comment:
we should test back compat upgrade + scale with this change (create cluster from current master + upgrade/scale from this branch)

Reply:
Since the agent SLB added in this PR won't already exists if users created a cluster from current master, scale and upgrade won't work. Issue #1190 currently cannot scale and upgrade with current master.

Reply:
Got it, in that case no need to test back compat 😅